draft-ietf-quic-transport.md

```diff
@@ -3797,17 +3797,23 @@ later time in the connection.
 The QUIC packet size includes the QUIC header and protected payload, but not the
 UDP or IP header.
 
+QUIC depends upon a minimum packet size of at least 1280 bytes.
+This is the IPv6 minimum size
+{{?RFC8200}} and is also supported by most modern IPv4 networks.
+Assuming the minimum IP header size, this results in a
+QUIC maximum packet size of 1232 bytes for IPv6 and 1252 bytes for IPv4. 
```

Comment:
I like clarity about constants and variables, the text in DPLPMTUD says: "The Maximum Packet Size (MPS) is the largest size of application data block that can be sent across a network path by a PL using a single Datagram.", so for QUIC, maybe this could read: 
"The QUIC maximum packet size is the largest size of QUIC packet that can be sent across a network path using a single packet. This is discovered using PMTUD/DPLPMTUD"?

Reply:
I could live with that.